dotcom-rendering/src/components/FollowButtons.test.tsx

```diff
@@ -31,18 +31,31 @@ it('should show a follow contributor button for a single contributor when render
 			displayName="Jon Doe"
 		/>,
 	);
-	expect(getByText('Follow Jon Doe')).toBeInTheDocument();
+	expect(getByText('Follow Jon Doe in My Guardian')).toBeInTheDocument();
 });
 
-it('should show a follow contributor button for a single contributor when rendering for apps', async () => {
+it('should not include the in My Guardian when the display name is longer than 21 characters in the follow action', () => {
+	const { getByText } = render(
+		<FollowTagButton
+			isFollowing={false}
+			onClickHandler={() => undefined}
+			displayName="abcdefghijklmnopqrstuv"
+		/>,
+	);
+	expect(getByText('Follow abcdefghijklmnopqrstuv')).toBeInTheDocument();
```

Comment:
This is an exceptionally basic heuristic to conditionally exclude the additional `in My Guardian` text when a tag is long enough to make it fall to a second line. Certainly not a perfect implementation, and open to suggestion on this, but as this is a small temporary workaround, this approach might be sufficient